src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -385,7 +385,9 @@ internal static PackageSpec ToPackageSpec(ProjectNames projectNames, IVsProjectR
                     RestoreAuditProperties = VSNominationUtilities.GetRestoreAuditProperties(targetFrameworks),
                 },
                 RuntimeGraph = VSNominationUtilities.GetRuntimeGraph(targetFrameworks),
-                RestoreSettings = new ProjectRestoreSettings() { HideWarningsAndErrors = true }
+                RestoreSettings = new ProjectRestoreSettings() { HideWarningsAndErrors = true },
+                SdkAnalysisLevel = VSNominationUtilities.GetSdkAnalysisLevel(targetFrameworks),
+                UsingMicrosoftNETSdk = VSNominationUtilities.GetUsingMicrosoftNETSdk(targetFrameworks)
```

Comment:
```suggestion
                UsingMicrosoftNETSdk = VSNominationUtilities.GetUsingMicrosoftNETSdk(targetFrameworks),
```

so any future properties only need to add lines, it won't need to modify an existing line